src/components/GameView/GameOverDialog.vue

```diff
@@ -90,6 +118,30 @@ export default {
 
       return 'loss-img';
     },
+    currentMatch() {
+      return this.game.currentMatch;
+    },
+    matchGameStats() {
+      const currentMatchGames = this.game.currentMatch?.games ?? [];
+      return currentMatchGames.map((game) => {
+        switch (game.result) {
```

Comment:
This is still pretty confusing, and it seems like a lookup map might do better, but as long as it works I'm ok with it for now. It feels like a refactor would be well warranted with better naming of some of the conditionals.

Reply:
It's tricky largely because the stored data is a bit funky to work with. It stores which user id was p0 and which was p1, and it stores whether p1 or p0 won, so then given the fact that p0 won (result = 0) you need to check if the current player was p0 or p1. I think it may make sense to revise the data architecture to store the user id of who won rather than the pNum of them (or maybe both)

Reply:
I think this function is reasonable given what's currently stored